test/components/testDateRngFmt.jsx

```diff
@@ -25,200 +25,139 @@ import Adapter from 'enzyme-adapter-react-16';
 import LocaleContext from '../../src/components/LocaleContext';
 import DateRngFmt from '../../src/components/DateRngFmt';
 import ilib from 'ilib-es6';
+import DateFactory from 'ilib-es6/lib/DateFactory';
 
 enzyme.configure({ adapter: new Adapter() });
 require("../assertExtras");
 
+const startDate = DateFactory({
+    year: 2019,
+    month: 6,
+    day: 27,
+    hour: 21,
+    minute: 54,
+    second: 33,
+    timezone: 'America/Los_Angeles'
+});
+
+const endDate = DateFactory({
+    year: 2019,
+    month: 7,
+    day: 2,
+    hour: 11,
+    minute: 21,
+    second: 6,
+    timezone: 'America/Los_Angeles'
+});
+
 export let testDateRngFmt = {
     testDateRngFmtSimple: test => {
         test.expect(1);
         const wrapper = mount(
             <LocaleContext.Provider value={{locale: "en-US"}}>
-                <DateRngFmt number={45.4}/>
+                <DateRngFmt length="short" start={startDate} end={endDate}/>
             </LocaleContext.Provider>
         );
 
-        test.equal(wrapper.text(), '45.4');
+        test.equal(wrapper.text(), '6/27/19 – 7/2/19');
         test.done();
     },
-
+    
     testDateRngFmtLocale: test => {
         test.expect(1);
         const wrapper = mount(
             <LocaleContext.Provider value={{locale: "en-US"}}>
-                <DateRngFmt locale="de-DE" number={453434.453}/>
-            </LocaleContext.Provider>
-        );
-
-        test.equal(wrapper.text(), '453.434,453');
-        test.done();
-    },
-
-    testDateRngFmtCurrency: test => {
-        test.expect(1);
-        const wrapper = mount(
-            <LocaleContext.Provider value={{locale: "en-US"}}>
-                <DateRngFmt locale="nl-NL" type="currency" currency="EUR" style="common" number={12345.4}/>
-            </LocaleContext.Provider>
-        );
-
-        test.equal(wrapper.text(), '€ 12.345,40');
-        test.done();
-    },
-
-    testDateRngFmtPercentage: test => {
-        test.expect(1);
-        const wrapper = mount(
-            <LocaleContext.Provider value={{locale: "en-US"}}>
-                <DateRngFmt locale="tr-TR" type="percentage" number={45.5}/>
+                <DateRngFmt locale="de-DE" length="short" start={startDate} end={endDate}/>
             </LocaleContext.Provider>
         );
 
-        test.equal(wrapper.text(), '%45,5');
+        test.equal(wrapper.text(), '27.06 – 02.07.19');
         test.done();
     },
 
-    testDateRngFmtScientific: test => {
+    testDateRngFmtCalendar: test => {
         test.expect(1);
         const wrapper = mount(
             <LocaleContext.Provider value={{locale: "en-US"}}>
-                <DateRngFmt style="scientific" number={453242.4}/>
+                <DateRngFmt calendar="hebrew" length="short" start={DateFactory({type: "hebrew", julianday: startDate.getJulianDay()})} end={DateFactory({type: "hebrew", julianday: endDate.getJulianDay()})}/>
```

Comment:
I'm failing this test case. react-ilib depend on the latest ilib (14.3.0) and ilib-es6 too. Is it related to my local timezone configuration? `Time zone: Asia/Seoul (KST, +0900)`
```
AssertionError: '3/25/79 – 3/30/79' == '3/25/79 – 3/29/79'
    at Object.equal (/home/goun/Source/opensource_iLib/react-ilib/node_modules/nodeunit/lib/types.js:83:39)
    at Object.equal (/home/goun/Source/opensource_iLib/react-ilib/test/components/testDateRngFmt.jsx:86:14)
```

When I change my timezone to `Time zone: America/Los_Angeles (PDT, -0700)` then it passed.

Reply:
Test is modified to include the time zone now.